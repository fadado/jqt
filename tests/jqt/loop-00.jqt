=======================
Looping blocks
=======================

{% range(2) %}
	2 lines
{% end %}
