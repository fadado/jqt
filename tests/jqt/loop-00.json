{
  "yes": true,
  "not": false,
  "nil": null,
  "string": "Neque porro quisquam",
  "number": 3.14159265358979323846,
  "array": [7,6,5,4,3,2,1 ],
  "object": { "b": true, "s": "Neque", "n": 3.14 }
}
