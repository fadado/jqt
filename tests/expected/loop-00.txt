=======================
Looping blocks
=======================

	2 lines
	2 lines
